timesketch/lib/analyzers/gcp_servicekey.py

```diff
@@ -0,0 +1,71 @@
+"""Sketch analyzer plugin for GCP Service Key usage."""
+from __future__ import unicode_literals
+
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+
+
+class GcpServiceKeySketchPlugin(interface.BaseSketchAnalyzer):
+    """Sketch analyzer for GCP Service Key usage."""
+
+    NAME = 'gcp_servicekey'
+
+    def __init__(self, index_name, sketch_id):
+        """Initialize The Sketch Analyzer.
+
+        Args:
+            index_name: Elasticsearch index name
+            sketch_id: Sketch ID
+        """
+        self.index_name = index_name
+        super(GcpServiceKeySketchPlugin, self).__init__(index_name, sketch_id)
+
+    def run(self):
+        """Entry point for the analyzer.
+
+        Returns:
+            String with summary of the analyzer result
+        """
+        query = ('principalEmail:*gserviceaccount*')
```

Comment:
Hm, can we make this query a bit cheaper? using "*" is a bit expensive. Dowe have a data_type attribute maybe?

Reply:
Unfortunately this version of parsed logs from dftimewolf has less fields to use compared to the previous one. I can remove one * from the query and make it "principalEmail:*gserviceaccount.com"

Reply:
Ok, no worries. lets keep as it is.